{"pytest_version": "7.2.2", "$report_type": "SessionStart"}
{"nodeid": "tests26_90_3.py::test_nums", "location": ["tests26_90_3.py", 14, "test_nums"], "keywords": {"test_nums": 1, "tests26_90_3.py": 1, "Folder_90": 1}, "outcome": "passed", "longrepr": null, "when": "setup", "user_properties": [], "sections": [], "duration": 0.00011589799942157697, "$report_type": "TestReport", "item_index": 2, "worker_id": "gw1", "testrun_uid": "8d804d13810e41e0ace66450aaab7dd7", "node": "<WorkerController gw1>"}
{"nodeid": "tests26_90_3.py::test_range_of_given_arguments", "location": ["tests26_90_3.py", 4, "test_range_of_given_arguments"], "keywords": {"test_range_of_given_arguments": 1, "tests26_90_3.py": 1, "Folder_90": 1}, "outcome": "passed", "longrepr": null, "when": "setup", "user_properties": [], "sections": [], "duration": 0.00016794500152172986, "$report_type": "TestReport", "item_index": 0, "worker_id": "gw0", "testrun_uid": "8d804d13810e41e0ace66450aaab7dd7", "node": "<WorkerController gw0>"}
{"nodeid": "tests26_90_3.py::test_range_of_given_arguments", "location": ["tests26_90_3.py", 4, "test_range_of_given_arguments"], "keywords": {"test_range_of_given_arguments": 1, "tests26_90_3.py": 1, "Folder_90": 1}, "outcome": "failed", "longrepr": {"reprcrash": {"path": "/home/shahin/Benchmark_Command/Q26/command_results_3/Folder_90/generated_answer.py", "lineno": 2, "message": "TypeError: unsupported operand type(s) for +: 'int' and 'list'"}, "reprtraceback": {"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    def test_range_of_given_arguments():", "        initial_list = list(range(273, 451 + 1))", ">       assert sum_in_range(initial_list) == sum(initial_list)"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q26/command_results_3/Folder_90/tests26_90_3.py", "lineno": 7, "message": ""}, "style": "long"}}, {"type": "ReprEntry", "data": {"lines": ["    def sum_in_range(nums):", ">       return sum(nums if n in range(273, 451 + 1) else 0 for n in nums)", "E       TypeError: unsupported operand type(s) for +: 'int' and 'list'"], "reprfuncargs": {"args": [["nums", "[273, 274, 275, 276, 277, 278, ...]"]]}, "reprlocals": null, "reprfileloc": {"path": "Q26/command_results_3/Folder_90/generated_answer.py", "lineno": 2, "message": "TypeError"}, "style": "long"}}], "extraline": null, "style": "long"}, "sections": [], "chain": [[{"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    def test_range_of_given_arguments():", "        initial_list = list(range(273, 451 + 1))", ">       assert sum_in_range(initial_list) == sum(initial_list)"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q26/command_results_3/Folder_90/tests26_90_3.py", "lineno": 7, "message": ""}, "style": "long"}}, {"type": "ReprEntry", "data": {"lines": ["    def sum_in_range(nums):", ">       return sum(nums if n in range(273, 451 + 1) else 0 for n in nums)", "E       TypeError: unsupported operand type(s) for +: 'int' and 'list'"], "reprfuncargs": {"args": [["nums", "[273, 274, 275, 276, 277, 278, ...]"]]}, "reprlocals": null, "reprfileloc": {"path": "Q26/command_results_3/Folder_90/generated_answer.py", "lineno": 2, "message": "TypeError"}, "style": "long"}}], "extraline": null, "style": "long"}, {"path": "/home/shahin/Benchmark_Command/Q26/command_results_3/Folder_90/generated_answer.py", "lineno": 2, "message": "TypeError: unsupported operand type(s) for +: 'int' and 'list'"}, null]]}, "when": "call", "user_properties": [], "sections": [], "duration": 0.00012805000005755574, "extra": [], "$report_type": "TestReport", "item_index": 0, "worker_id": "gw0", "testrun_uid": "8d804d13810e41e0ace66450aaab7dd7", "node": "<WorkerController gw0>"}
{"exitstatus": 2, "$report_type": "SessionFinish"}
